src/Apps/Conversation/Components/Conversations.tsx

```diff
@@ -13,7 +14,7 @@ const Conversations: React.FC<ConversationsProps> = props => {
   const { me } = props
   const conversations = me.conversationsConnection.edges
   return (
-    <Box px={1}>
+    <Box px={1} height="calc(100vh - 180px)">
```

Comment:
Does this `180` represent the height of something? Can this value be shared across where it's used so that things remain in sync?

Reply:
So, this is non-ideal. There's an issue w/ the height of this component generally and how it plays with the header that's causing all sorts of headache. I'd like to leave this for now, but when I tackle the desktop layout I'm going to revisit this and clean it up.